docs/reference/samples/observability.filters-pipelines-unchanged.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: "observability.openshift.io/v1"
```

Suggest to change to unquoted:

```suggestion
apiVersion: observability.openshift.io/v1
```